pack/rpm.mk

```diff
@@ -109,7 +109,7 @@ $(BUILDDIR)/$(RPMSRC): $(BUILDDIR)/$(TARBALL) \
 		--define '_sourcedir $(BUILDDIR)' \
 		--define '_specdir $(BUILDDIR)' \
 		--define '_srcrpmdir $(BUILDDIR)' \
-		--define '_ci $(CI)' \
+		--define '_ci "$(CI)"' \
```

Comment:
There are not so many languages that looks more disgusting then bash. RPM spec language earns the prize.

The quotes are literally included into the macro definition and so `%if "%{_ci}" == "travis"` would fail. I also see that usual 'no value' value is `%{nil}` macro, which expands to nothing. I would propose the following way:

```
ifeq ($(CI),)
       CI_MACRO=%{nil}
else
       CI_MACRO=$(CI)
endif
```

What do you think?

Reply:
I've checked and it works, thanks !